privilege/privileges/privileges.go

```diff
@@ -32,11 +32,13 @@ var SkipWithGrant = false
 const (
 	codeInvalidPrivilegeType  terror.ErrCode = 1
 	codeInvalidUserNameFormat                = 2
+	codeErrNonexistingGrant                  = 1141
 )
 
 var (
```

Comment:
Should we extract these lines and `init()` function to an individual file privilege/privileges/errors.go?